public/app/features/alerting/components/PreviewQueryTab.tsx

```diff
@@ -24,11 +26,19 @@ export const PreviewQueryTab: FC<Props> = ({ data, height, width }) => {
   // Select padding
   const padding = 16;
 
-  if (!data?.series?.length) {
-    return <Table data={toDataFrame([])} height={height} width={width} />;
+  if (!data) {
+    return (
+      <EmptyState title="Run queries to view data.">
+        <Button onClick={onRunQueries}>Run queries</Button>
+      </EmptyState>
+    );
   }
 
-  if (data.series.length > 1) {
+  if (!data.series) {
+    return null;
+  }
+
+  if (data.series?.length > 1) {
```

Comment:
Nit: no need for `?` here right?
```suggestion
  if (data.series.length > 1) {
```